inference-engine/tests/functional/plugin/myriad/single_layer_tests/swish.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <functional_test_utils/layer_test_utils.hpp>
+#include <ngraph_functions/utils/ngraph_helpers.hpp>
+#include <ngraph_functions/builders.hpp>
+
+#include <tuple>
+#include <string>
+
+using Parameters = std::tuple<
+        ngraph::element::Type,
+        ngraph::Shape,
+        LayerTestsUtils::TargetDevice>;
+
+namespace LayerTestsDefinitions {
+class SwishLayerTest : public testing::WithParamInterface<Parameters>,
```

Comment:
Could you explain, what different between this test and test from the `activation.cpp` file ?

Reply:
Nothing. Thx, just a mistake.

Reply:
Fixed